areas/appservice/tests/test-resources.bicep

```diff
@@ -0,0 +1,377 @@
+targetScope = 'resourceGroup'
+
+@minLength(3)
+@maxLength(17)
+@description('The base resource name.')
+param baseName string = resourceGroup().name
+
+@description('The location of the resource. By default, this is the same as the resource group.')
+param location string = resourceGroup().location
+
+@description('The client OID to grant access to test resources.')
+param testApplicationOid string
+
+@description('SQL Server administrator login name.')
+param sqlAdminLogin string = 'mcptestadmin'
+
+@description('SQL Server administrator password.')
+@secure()
+param sqlAdminPassword string = newGuid()
+
+@description('MySQL administrator login name.')
+param mysqlAdminLogin string = 'mcptestadmin'
+
+@description('MySQL administrator password.')
+@secure()
+param mysqlAdminPassword string = newGuid()
+
+@description('PostgreSQL administrator login name.')
+param postgresAdminLogin string = 'mcptestadmin'
+
+@description('PostgreSQL administrator password.')
+@secure()
+param postgresAdminPassword string = newGuid()
```

Comment:
This list is missing Cosmos

Reply:
I removed it, since I didn't want to add every single database instance to test it. I think a few types are enough